cmd/bio-rd/main.go

```diff
@@ -37,6 +37,8 @@ var (
 func main() {
 	flag.Parse()
 
+	log.SetLevel(log.DebugLevel)
```

Comment:
Did you want to commit this? :)

Reply:
Sure, as long as this is under development and far away from a beta release.